Makefile

```diff
@@ -84,27 +92,30 @@ Q = $(if $(filter 1,$V),,@)
 H = $(shell printf "\033[34;1m=>\033[0m")
 
 .PHONY: default
-default: depend build
+default: depend build test
```

Comment:
IMHO the default target should be just a help, with the list of common targets and explanations :-)
But build test seems ok too